app/services/foreman/foreman_url_renderer.rb

```diff
@@ -39,20 +37,44 @@ def foreman_url(action, params = {}, unescaped_params = {})
         raw_string += unescaped_params.map { |k, v| "#{k}=#{v}" }.join('&')
       end
 
+      url_options[:action] = action
+      url_options[:path] = config.path
+      render_foreman_url(host, url_options, params) + raw_string
+    end
+
+    apipie :method, 'Returns Foreman request address' do
```

Comment:
request address is a new term and I had an hard time to connect it to the `foreman_url`.
Would it be worth noting that this is related to unattended url?
the description for foreman_url: `Returns Foreman unattended URL`, should this be `Returns address part of Foreman unattended URL`?